### ISSUE TITLE
Default drag-handle

### ISSUE THREAD

**Issue by njordhov** [Kah0ona/re-dnd#1]
**Thank you for developing this widget!** I used your brand new release today to implement an orderable list.

I have a small suggestion: Provide a catch-all `:default` for `e-dnd.views/drag-handle` that covers the typical case. That way, implementing a custom drag-handle method may not be necessary. 

```clojure
(defmethod drag-handle
  :default
  [{:keys [type id]}]
  [:div.drop-marker])
```

**Comment by Kah0ona** [comment]
Oh my... somehow I completely missed all these conversations. Something with an e-mail rule in my gmail filtering these out. 

Thanks for your PR, I merged it.

Will do this.

**Comment by Kah0ona** [comment]
Fixed in 0.1.11
